lms/djangoapps/discussion_api/api.py

```diff
@@ -647,9 +649,11 @@ def update_thread(request, thread_id, update_data):
     # Only save thread object if some of the edited fields are in the thread data, not extra actions
     if set(update_data) - set(actions_form.fields):
         serializer.save()
+        # signal to update Teams information if updated
```

Comment:
Please update the comment.  The signal that is fired is a generic signal.  Currently, the Teams code may be the only subscriber to this signal.  But soon, we'll have other subscribers as well (for example, the Notifications app).